presto-hive-hadoop2/bin/run_hive_s3_tests.sh

```diff
@@ -24,6 +24,14 @@ exec_in_hadoop_master_container hadoop fs -copyFromLocal -f /tmp/test1.csv.lz4 "
 exec_in_hadoop_master_container hadoop fs -copyFromLocal -f /tmp/test1.csv.bz2 "${table_path}"
```

Comment:
I think to actually fix https://github.com/prestodb/presto/issues/12153, you would need revert the addition of `/tmp/test1.csv.*` here, into the `s3a://${S3_BUCKET}/presto_test_external_fs/` location

Reply:
Additional `test1.csv.*` files under `s3a://${S3_BUCKET}/presto_test_external_fs/` represent additional duplicate rows in `presto_test_external_fs` table. As all of the tests pass locally, I believe, they shouldn't be asserting on the exact content of the file, except for `testGetRecords` that we have already updated. 

Ideally, a table could consist of multiple objects and thus the tests should be resilient of the change of adding a new supported object. Also, the end-to-end happy case tests that are supported by S3 SELECT should always be supported by S3 GET.  Am I missing something over here? Should we fix the specific tests that couldn't handle multiple files instead?

Reply:
@electrum, @findepi, @kokosing Do we want to address `%RANDOM_PART%` as a part of this PR or should we create an issue instead and close this PR?

Reply:
I would close this one and create another issue and PR.

Reply:
Sure will close this PR then.

Reply:
I would not create another issue. 
https://github.com/prestodb/presto/issues/12153 got addressed internally and closed, and i don't see a need for more changes currently.